test/common/router/BUILD

```diff
@@ -386,3 +388,24 @@ envoy_proto_library(
         "@envoy_api//envoy/extensions/filters/http/router/v3:pkg",
     ],
 )
+
+envoy_cc_benchmark_binary(
+    name = "config_impl_speed_test",
+    srcs = ["config_impl_speed_test.cc"],
```

Comment:
There's an open conversation about name of these targets and source files in #13327 

speed_test makes sense for consistency with current sources, but calling them benchmark and benchmark_test may be better long term.  Let's see what we decide on that other PR.

Reply:
Sure, I'll wait for a few days.